src/main/java/oc/wh40k/units/im/IMUriahJacobus.java

```diff
@@ -1,38 +1,41 @@
 package oc.wh40k.units.im;
 
-
+import static oc.KeyWord.*;
 
 import oc.Eintrag;
 
-
-
 public class IMUriahJacobus extends Eintrag {
 
-
-
     public IMUriahJacobus() {
-
+    	super(IMPERIUM, ADEPTUS_MINISTORUM, CHARACTER, INFANTRY, MINISTORUM_PRIEST, MISSIONARY, URIAH_JACOBUS);
         name = "UriahJacobus";
-
         grundkosten = getPts("UriahJacobus");
+        power = 3;
 
-        power = 5;
-
-
+        seperator();
+        
+        addWarlordTraits("Righteous Rage", true);
 
+        addToInformationVector("MINISTORUM_PRIEST", 1);
```

Comment:
Constants for String literals. Reuse the ones created for other Eintrag classes.